README.md

```diff
@@ -1,19 +1,161 @@
+                number for the log level verbosity
+      --vmodule pattern=N,...                                                                                                                                                                                          
+                comma-separated list of pattern=N settings for file-filtered logging (only works for text log format)
+
+Common flags:
+
+      --feature-gates mapStringBool                                                                                                                                                                                    
+                A set of key=value pairs that describe feature gates for alpha/experimental features. Options are:
+                AllAlpha=true|false (ALPHA - default=false)
+                AllBeta=true|false (BETA - default=false)
+                ContextualLogging=true|false (ALPHA - default=false)
+                LoggingAlphaOptions=true|false (ALPHA - default=false)
+                LoggingBetaOptions=true|false (BETA - default=true)
+      --version                                                                                                                                                                                                        
+                print binary version and exit
+
+Controller flags:
+
+      --config-name string                                                                                                                                                                                             
+                The name of the ConfigMap which holds controller configuration (default "nvidia-k8s-ipam-config")
+      --config-namespace string                                                                                                                                                                                        
+                The name of the namespace where ConfigMap with controller configuration exist (default "kube-system")
+      --health-probe-bind-address string                                                                                                                                                                               
+                The address the probe endpoint binds to. (default ":8081")
+      --kubeconfig string                                                                                                                                                                                              
+                Paths to a kubeconfig. Only required if out-of-cluster.
+      --leader-elect                                                                                                                                                                                                   
```

Comment:
I thought we don't support it. Maybe we should document only the things that will work

Reply:
In the code it is supported. 
I added [1] extra config option to configure namespace for leader election process. Also enabled leader election by default to protect from the situation when a user decided to manually scale the deployment. 

[1] https://github.com/Mellanox/nvidia-k8s-ipam/pull/19

Reply:
These are the supported flags by the binary, should i remove or keep ?

either way #19 would need to update this file as well once we merge it.